doc/source/main-documentation/qibolab.rst

```diff
@@ -0,0 +1,611 @@
+Platforms
+---------
+
+Qibolab provides support to different quantum laboratories.
+
+Each lab is implemented using a :class:`qibolab.platform.Platform` object which implements basic features and connects instruments, qubits and channels.
+Therefore, the ``Platform`` enables the user to interface with all
+the required lab instruments at the same time with minimum effort.
+
+In the API reference section, a description of all the attributes and methods of the ``Platform`` is provided. here, let's focus on the main elements.
+
+In the platform, the main methods can be divided in different sections:
+
+- functions to load qubit parameters, save them and change them (``reload_settings``, ``dump``, ``update``)
+- functions to coordinare the instruments (``connect``, ``setup``, ``start``, ``stop``, ``disconnect``)
+- functions to execute experiments (``execute_pulse_sequence``, ``execute_pulse_sequences``, ``sweep``)
+- functions to initialize gates (``create_RX90_pulse``, ``create_RX_pulse``, ``create_CZ_pulse``, ``create_MZ_pulse``, ``create_qubit_drive_pulse``, ``create_qubit_readout_pulse``, ``create_RX90_drag_pulse``, ``create_RX_drag_pulse``)
+- setter and getter of channel/qubit parameters (local oscillator parameters, attenuations, gain and biases)
+
+The idea of the ``Platform`` is to serve as the only exposed object to the user so, for example, we can easily write an example of experiment, whithout any need of going into the low-level instrument-dpecific code.
+
+For example, let's first define a platform (that we consider to be a single qubit platform) using the ``create`` method presented in :doc:`/tutorials/lab`:
+
+.. code-block::  python
+
+    from qibolab import Platform
+    platform = Platform("my_platform")
+
+Now we connect and start the instruments (note that we, the user, do not need to know which instruments are connected).
+
+.. code-block::  python
+
+    platform.connect()
+    platform.setup()
+    platform.start()
+
+We can easily print some of the parameters of the channels (similarly we can set those, if needed):
+
+.. note::
+   If the get_method does not apply to the platform (for example there is no local oscillator, to TWPA or no flux tunability...) a ``NotImplementedError`` will be raised.
+
+.. code-block::  python
+
+    print(f"Drive LO frequency: {platform.get_lo_drive_frequency(0)}")
```

Comment:
My idea was to replace these methods with the qubit/channel API. So instead of
```py
platform.get_lo_drive_frequency(0)
```
use
```py
platform.qubits[0].drive.lo_frequency
```
but I didn't do it yet, as it would require changing qibocal too. 

Since currently both exist it is fine to use either in the documentation. I am in favor of the latter, but that's my personal opinion which we don't necessarily need to follow.